test/__mocks__/redis.js

```diff
@@ -57,17 +59,17 @@ class Redis {
         }
     }
 
-    hget(key, internalKey) {
+    hgetall(key) {
         const obj = this.get(key);
-        if (obj === null) {
-            return null;
-        }
-        return obj[`${internalKey}`];
+        return obj;
     }
     multi() {
         return { hmset: this.hmset, set: this.set }
     };
     hmset(key, value) {
+        if (setCallKeysForError.includes(key)) {
+            throw new Error("Connection is Closed");
```

Comment:
Change all `Error` to `PlatformError`

Reply:
It is just used for mocking.
Also if some `errorType` should be given to that I guess it should be `redisError`

Reply:
Do we have that class, use should use the error classes already defined so that we get alerted correctly

Reply:
Yes we have that class and for redis alerts we are currently handling at destination level using metrics. [Ref](https://github.com/rudderlabs/rudderstack-operator/pull/1547/files)